### PR TITLE
Feature/create more GitHub issue/pr template #518

### PR DESCRIPTION
**Pull request by jorgelaranjo**
## 🎯 Changes

Adds

.github/PULL_REQUEST_TEMPLATE/add_update_component.md
.github/PULL_REQUEST_TEMPLATE/delete_component.md
.github/PULL_REQUEST_TEMPLATE/other_changes.md
.github/ISSUE_TEMPLATE/request_change.md

Also makes old template available on new folder and keeps a copy on the current location not to break functionality as-is

as per #518 request

## ✅ Checklist

- [X] I have given my PR a descriptive title
- [X] I have run `pnpm run lint` locally on my changes

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATES/add_update_component.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+<!-- Describe the changes made in this PR. Is it a new component, tool, or template? Or is it an update to an existing one? -->
+
+## 📝 Description
+
+<!-- Provide a detailed description of the changes. What problem does it solve? How does it improve the existing functionality? -->
+
+## 📸 Screenshots (if applicable)
+
+<!-- Include screenshots of the new or updated component/tool/template if applicable. -->
```
Comment (MacFJA) [inline .github/PULL_REQUEST_TEMPLATES/add_update_component.md:11]: I don't know about this section.
We don't have a screenshot on packages or templates.

File: .github/PULL_REQUEST_TEMPLATES/add_update_component.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## 🎯 Changes
+
+<!-- Describe the changes made in this PR. Is it a new component, tool, or template? Or is it an update to an existing one? -->
```
Comment (MacFJA) [inline .github/PULL_REQUEST_TEMPLATES/add_update_component.md:3]:
```suggestion
<!-- Describe the changes made in this PR. Is it a new package or template? Or is it an update to an existing one? -->
```
The distinction between component and tool have been removed.
There are only 6 kind of content (and 2 for external source code):
- package (before it was component and tools)
- template
- recipe
- cheat sheet
- event
- resource (video, books)

File: .github/PULL_REQUEST_TEMPLATES/delete_component.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+## 🗑️ Deletion Request
+
+<!-- Describe the component/template/tool that is being deleted and the reasons for its removal. -->
```
Comment (MacFJA) [inline .github/PULL_REQUEST_TEMPLATES/delete_component.md:3]:
```suggestion
<!-- Describe the package/template that is being deleted and the reasons for its removal. -->
```

**Comment by kevmodrome** [comment]
This is great for the current site, but as we're gearing up to release the new site at the end of the month, I think it's unnecessary to implement. We'll be using a submission system on the actual site to add things.

**Pull request closed without merging**